packages/atomic/src/mixins/light-dom.ts

```diff
@@ -0,0 +1,106 @@
+import type {
+  CSSResultArray,
+  CSSResultGroup,
+  CSSResultOrNative,
+  LitElement,
+} from 'lit';
+import type {Constructor} from './mixin-common.js';
+
+declare class LightDomMixinInterface {
+  injectStyles(dynamicStyles?: CSSResultGroup): Promise<void>;
+}
+
+/**
+ * Mixin that provides dynamic styles injection functionality for components that do not use Shadow DOM.
+ *
+ * This mixin replaces the deprecated `@injectStylesForNoShadowDOM` decorator and is intended for LitElement-based
```

Comment:
In fact you're deleting the decorator entirely, not just deprecating it!

You should also adjust the JSDoc in bindings.ts for the AdoptedStylesBindings interface.